.github/workflows/lint.yaml

```diff
@@ -29,3 +29,26 @@ jobs:
 
       - name: Pint
         run: vendor/bin/pint --test
+  phpstan:
```

Comment:
Can you also rename the existing "Lint" job to "Pint" and change the php to 8.3 too?